admin_guide/managing_networking.adoc

```diff
@@ -275,6 +275,15 @@ above example, suppose `www.foo.com` resolved to `10.11.12.13` and has a DNS TTL
 of one minute, but was later changed to `20.21.22.23`. {product-title} will then
 take up to one minute to adapt to these changes.
 
+[NOTE]
+====
+The default installer always allows pods access to the external interface of the 
```

Comment:
How about:

The egress firewall always allows pods access to the external interface of the node the pod is on. If your DNS resolution is not handled by something on the local node, the you will need to add egress firewall rules allowing access to the DNS server's IP addresses if you are using domain names in your pods.  The default installer sets up a local dnsmasq, so if you are using that setup you will not need to add extra rules.

Please add a link around "default installer" to the right docs for the installer.